### PR TITLE
feat: add Integrity and Hygiene ratings if present

### LINKED ISSUE #123: FEAT: Show Popularity, Hygiene and Integrity Rating for Components
Can we add Component Metadata as Popularity, Hygiene and Integrity Rating to the extension? 

This will help developers in their selections of best components in the central repositories. 

A nice place to display would be right under the component name in the **Info** tab.

### PR DESCRIPTION
**Pull request by maurycupitt**
Add Integrity and Hygiene ratings if present.

Resolves #123 

Both Integrity and Hygiene: https://central.sonatype.com/artifact/org.apache.struts/struts2-core/6.1.2
Hygiene Only: https://central.sonatype.com/artifact/org.apache.struts/struts2-core/2.5.29
Integrity Only: https://www.npmjs.com/package/express/v/4.19.2

### CODE REVIEW

**Comment by madpah** [comment]
We need to add message entries for the other languages @maurycupitt

File: public/_locales/en/messages.json
Comment (madpah) [inline public/_locales/en/messages.json:1]: We need to add translations.

**Review by madpah — CHANGES_REQUESTED** [review]

File: public/_locales/en/messages.json
Comment (maurycupitt) [reply public/_locales/en/messages.json:1]: done

**Review by madpah — APPROVED** [review]

**Comment by madpah** [comment]
Merge on hold until we have reviewed some account validation settings with Google and Microsoft @maurycupitt.

**Pull request merged**